test/simulation/config.go

```diff
@@ -5,7 +5,7 @@ import "time"
 const (
 	defaultXudKill    = true
 	defaultXudCleanup = true
-	defaultTimeout    = int64(15 * time.Second)
+	defaultTimeout    = int64(300 * time.Second)
```

Comment:
How did you come to 300 seconds, trial and error? It looks like it would still make sense to go through the simulation tests after this PR is merged and try to tidy up usage of contexts and context timeouts, does that sound right to you?